reactDashboard/src/pages/Regional.jsx

```diff
@@ -47,6 +47,23 @@ const Regional = ({
 
   const currentRegionCode = useRef(regionCode);
   const currentLocation = useRef(match.url);
+  const [dateRange, setDateRange] = useState("dateRange");
+  const [maxDateRange, setMaxDateRange] = useState({
+    startDate: 0,
+    endDate: 0,
+  });
+
+  useEffect(() => {
+    // Only attempt to fetch data once
+    if (healthBoardDataset != null || councilAreaDataset != null) {
```

Comment:
Same issue here

Reply:
The issue here was needing if (healthBoardDataset != null) { to prevent a race condition bug